specification/paths/Payments-v1-providers-code-connect.json

```diff
@@ -36,6 +36,16 @@
                     "format": "uuid",
                     "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
                     "example": "4067e35b-09cc-42b9-8553-6eaf1788df35"
+                  },
+                  "settings": {
+                    "type": "object",
+                    "description": "The settings to use for the connection. See the payment provider's settings_format for details."
+                  },
```

Comment:
We could add an `"example"` here, so the preview panel does not just show `"settings": { },`. We do the same for our shop-integration routes. I'm also fine leaving it like this, since it's an optional attribute.
What do yo think?

Reply:
Example will be very useful! I will add one!